modules/ui/view_on_maproulette.js

```diff
@@ -0,0 +1,46 @@
+import { uiIcon } from './icon';
+import { Task } from '../maproulette/Task';
+
+
+export function uiViewOnMapRoulette(context) {
+  const l10n = context.systems.l10n;
+  const maproulette = context.services.maproulette;
+  let _task;
+
+
+  function viewOnMapRoulette(selection) {
+    let url;
+    if (maproulette && (_task instanceof Task)) {
+      url = maproulette.itemURL(_task);
```

Comment:
@Bonkles I have a question about `itemURL` here: I've already modified it to return a maproulette link, however, I see no link on the left side panel. Why is that?